### ISSUE TITLE
RTP payload type incorrectly set via REST API

### ISSUE THREAD

**Issue by f4exb** [f4exb/sdrangel#526]
When trying to change the audio RTP payload type remotely using the REST API the actual payload type passed is always 0. This results in broken audio link when the client expects a format different from the default L16 mono.

Here trying to change to PCMU8 (index 4 of the enum so should be 4)

```
2020-05-23 12:12:02.846 (D) HttpConnectionHandler (0xaaab00279550): read timeout occurred
2020-05-23 12:12:02.849 (D) HttpConnectionHandler (0xaaab00279550): disconnected
2020-05-23 12:12:18.492 (D) HttpConnectionHandler (0xaaab00279550): received request from 192.168.0.3 (OPTIONS) /sdrangel/audio/output/parameters
2020-05-23 12:12:18.493 (D) WebAPIRequestMapper::service: method OPTIONS: assume pre-flight
2020-05-23 12:12:18.500 (D) HttpConnectionHandler (0xaaab00279550): received request from 192.168.0.3 (PATCH) /sdrangel/audio/output/parameters
2020-05-23 12:12:18.501 (D) AudioNetSink::setParameters:  codec:  0  stereo:  false  sampleRate:  48000
2020-05-23 12:12:18.501 (D) AudioFilter::calculate2:  highPass:  false  fc:  0.45  a0:  1  a1:  -1.55627  a2:  -0.644171  b0:  0.80011  b1:  1.60022  b2:  0.80011
2020-05-23 12:12:18.501 (D) AudioFilter::calculate2:  highPass:  true  fc:  0.00104167  a0:  1  a1:  1.99081  a2:  -0.990853  b0:  0.995415  b1:  -1.99083  b2:  0.995415
2020-05-23 12:12:18.501 (D) RTPSink::setPayloadInformation: payloadType: 0 sampleRate: 48000
2020-05-23 12:12:18.502 (D) RTPSink::setPayloadInformation: timestamp unit set to 0.000021: No error
2020-05-23 12:12:18.502 (D) RTPSink::setPayloadInformation: set default mark to false: No error
2020-05-23 12:12:18.502 (D) RTPSink::setPayloadInformation: set default timestamp increment to 960: No error
2020-05-23 12:12:18.502 (D) RTPSink::setPayloadInformation: set maximum packet size to 1940 bytes: No error
2020-05-23 12:12:18.502 (D) AudioNetSink::setDecimation:  1  from:  1
2020-05-23 12:12:18.503 (D) AudioFilter::calculate2:  highPass:  false  fc:  0.45  a0:  1  a1:  -1.55627  a2:  -0.644171  b0:  0.80011  b1:  1.60022  b2:  0.80011
2020-05-23 12:12:18.503 (D) AudioFilter::calculate2:  highPass:  true  fc:  0.00104167  a0:  1  a1:  1.99081  a2:  -0.990853  b0:  0.995415  b1:  -1.99083  b2:  0.995415
2020-05-23 12:12:18.504 (D) AudioDeviceManager::setOutputDeviceInfo: index: -1 device: System default device updated
2020-05-23 12:12:28.847 (D) HttpConnectionHandler (0xaaab00279550): read timeout occurred
2020-05-23 12:12:28.848 (D) HttpConnectionHandler (0xaaab00279550): disconnected
```
Relevant line shows type "0" this is always the case no matter what is passed through REST API: `2020-05-23 12:12:18.501 (D) RTPSink::setPayloadInformation: payloadType: 0 sampleRate: 48000`
Edit: Error further above: `AudioNetSink::setParameters:  codec:  0  stereo:  false  sampleRate:  48000` (the client SDRangelCli may be checked also).

**Comment by f4exb** [comment]
Issue is in the server web adapter `WebAPIAdapterSrv::instanceAudioOutputPatch` the GUI one `WebAPIAdapterGUI::instanceAudioOutputPatch` is correct. 

`WebAPIAdapterSrv::instanceAudioOutputPatch` is missing these lines:

```c++
    if (audioOutputKeys.contains("udpChannelCodec")) {
        outputDeviceInfo.udpChannelCodec = static_cast<AudioOutput::UDPChannelCodec>(response.getUdpChannelCodec());
    }
    if (audioOutputKeys.contains("udpDecimatiobFactor")) {
        outputDeviceInfo.udpDecimationFactor = response.getUdpDecimationFactor();
    }
```

**Comment by f4exb** [comment]
Implemented in v5.6.2 and v4.14.8
